.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -59,6 +59,22 @@ parameters:
     displayName: Branch in aks-rp to use for overrides
     type: string
     default: master
+  - name: ws2019BaseImageUrl
```

Comment:
We keep these variables as secret and do not want to expose the values in any logs.

Reply:
I think it's OK to expose them in the devops logs as they are only accessible internally. Also, anyone who has access to the pipelines can view the variables or parameters that were used to run the pipelines - so I'm not sure we gain much by using variables.